packages/algob/src/lib/account.ts

```diff
@@ -98,11 +98,14 @@ export function loadAccountsFromEnv (): Account[] {
 }
 
 // returns multisignature account address
-export function createMsigAddress (version: number, threshold: number, accountList: AccountSDK[]): any {
+export function createMsigAddress (
+  version: number,
+  threshold: number,
+  accountList: string[]): [MultiSigAccount, string] {
   const mparams = {
     version: version,
     threshold: threshold,
-    addrs: accountList
+    address: accountList
```

Comment:
shouldn't this be `addrs`/`addresses`? It's a list of addresses from which we create the msig hash